draft-ietf-taps-interface.md

```diff
@@ -1413,9 +1413,42 @@ Remote Endpoint, and also the transport properties and security parameters
 needed for Protocol Stack selection.
 
 The Rendezvous() Action causes the Preconnection to listen on the Local
-Endpoint for an incoming Connection from the Remote Endpoint, while
-simultaneously trying to establish a Connection from the Local Endpoint to the
-Remote Endpoint. This corresponds to a TCP simultaneous open, for example.
+Endpoint for an incoming Connection from the Remote Endpoint, while also
+simultaneously trying to establish a Connection from the Local Endpoint to
+the Remote Endpoint. 
+
+If there are multiple local or remote endpoints configured, then a rendezvous
+action will systematically probe reachability of those endpoints following the
+approach used in Interactive Connectivity Establishment (ICE) {{?RFC5245}}.
```

Comment:
Do we need to specify this this tightly? (I think it's safe to do so, ICE is an approach, not a protocol. But we've pointedly avoided to make tight bindings to a Single Way Of Doing Things, so we should make this choice explicitly)

Reply:
also, if we're specifying ICE, this needs to be {{!RFC5245}}.

Reply:
Will rephrase to be slightly more general: "If there are multiple local or remote endpoints configured, then a rendezvous action will need to systematically probe reachability of those endpoints following an approach such as that used in Interactive Connectivity Establishment (ICE) {{?RFC5245}}."